### DISCUSSION TITLE
Notice subtypes E1, E2, E3, E4 and E5

### DISCUSSION THREAD

**Discussion by tamer-erdogan** [OP-TED/eForms-SDK#244] [Q&A] [answered]
The subtypes E1, E2, E3, E4, E5 are not found in the notice-types.json.
But interestingly they exist in the translations file and also in the CELEX_32019R1780_EN_ANNEX_TABLE2_Extended.xlsx
Could you please add these types also.

**Comment by bertrand-lorentz** [comment, marked as answer]
Thanks for your feedback.
Those subtypes are currently not implemented. We do have some information about them, which is why you might see some references to them in the eForms SDK.
Please see the answer to question 13 "What is meant by E1, E2, E3, E4 and E5 in the Excel document annexed to the eForms regulation?" in the section "Forms and procedures" of our FAQ on our documentation website, at https://docs.ted.europa.eu/home/eforms/FAQ/index.html#_forms_and_procedures

As this is not an issue in the SDK, I'm moving this to the forum under "Discussions", so that it can be seen by others.

**Reply by tamer-erdogan** [reply]
Do you think these Extended Definitions will be defined in the notice-types folder? Our implementation depends on the json files in the notice-types folder. Otherwise we cannot show the Extended types.

**Reply by bertrand-lorentz** [reply]
Yes, when those subtypes are implemented, the corresponding files will be added in the "notice-types" folder. Our own application (eNotices2) uses those files for the layout of the form on the screen, so we will need them also.

**Reply by jlouvrier** [reply]
@bertrand-lorentz Any ETA regarding implementation of those subtypes?

**Reply by bertrand-lorentz** [reply]
From what I know, the notice subtypes E1 to E5 should be part of the next amendment to the eForms regulation, which is currently foreseen for the end of 2023. So we should have them implemented around that time.
